api/v1alpha1/secretagentconfiguration_types.go

```diff
@@ -112,9 +112,10 @@ const (
 
 // SecretsManager Strings
 const (
-	SecretsManagerNone SecretsManager = "none"
-	SecretsManagerGCP  SecretsManager = "GCP"
-	SecretsManagerAWS  SecretsManager = "AWS"
+	SecretsManagerNone  SecretsManager = "none"
+	SecretsManagerGCP   SecretsManager = "GCP"
+	SecretsManagerAWS   SecretsManager = "AWS"
+	SecretsManagerAzure                = "Azure"
```

Comment:
```suggestion
	SecretsManagerAzure  SecretsManager  = "Azure"
```
This should be of type `SecretsManager` in order to pass validation.